geopandas/io/file.py

```diff
@@ -48,6 +50,9 @@ def _import_fiona():
                 except ImportError:
                     fiona_env = None
 
+            FIONA_GE_19 = Version(Version(fiona.__version__).base_version) >= Version(
```

Comment:
Is there a specific reason the `.base_version` is needed here?

Reply:
Ah, to deal with the fact that 1.9a1 is smaller than 1.9?

Reply:
Yeah - essentially to ensure any pre-release version was included.